### DISCUSSION TITLE
Invalid application for farming rewards

### DISCUSSION THREAD

**Discussion by a397184189** [Chia-Network/chia-blockchain#15724] [Support]
The reward has been applied for 2 days but has not been received yet. The bill shows that it is currently being processed

Solo Cooperative Address xch1aecx2rqx6l0cln7e5rfup4aslap47ypxj0wvwu2xw6slxggutqsa2alze
![image](https://github.com/Chia-Network/chia-blockchain/assets/19707344/fcda793e-7dc4-431c-9ab2-371c944d7e68)
![image](https://github.com/Chia-Network/chia-blockchain/assets/19707344/fc0bb51b-c8bb-4d12-8eeb-67b2e7daccbd)

![image](https://github.com/Chia-Network/chia-blockchain/assets/19707344/4e8f6d5f-be17-4dc3-ace2-bff9def63ca5)

**Comment by BrandtH22** [comment]
Good afternoon @a397184189 , I am sorry for the delayed response, much of our support is currently focused on discord (https://discord.gg/chia) but we recognize that more attention needs to be given to github issues and are working to do so.

I am a bit unclear as to the question and will need a bit of clarification.
It seems you are solo pooling and have a balance that can be claimed, is it that the claim solo rewards option is not claiming the rewards properly?

**Reply by Zerotree-ruyi** [reply]
我也是这样   领取不了

**Reply by Zerotree-ruyi** [reply]
提示双重交易

**Reply by Zerotree-ruyi** [reply]
别人说的很清楚 还有图片 还要怎么解释
